src/server/tasks/npm-install-task.ts

```diff
@@ -14,6 +14,7 @@ export class NpmInstallTask extends ConsoleTask<void> {
    * Boots the webpack dev server. Returns
    */
   protected async spawnChildProcess(): Promise<[void, ChildProcess]> {
+    process.env.NODE_ENV = 'development';
     const childProcess = spawn('npm', ['install', '-d', '--color=always'], {
       cwd: this.directory,
       env: process.env,
```

Comment:
Pass `{ ...process.env, NODE_ENV: 'development' }` in here instead. Mutating the global process state, particularly in this random method, is a bit icky. Functions should avoid having side effects when possible, and when side effects are unavoidable, only effect those that are well-documented and expected.

Reply:
Done 😄